### PR TITLE
Add version configurations

### PR DESCRIPTION
**Pull request by yixinglu**
depends on vesoft-inc/nebula-common#373

This PR aims at decreasing the effect of the macro of GIT_INFO_SHA for ccache

image

### CODE REVIEW

File: src/version/Version.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
```
Comment (laura-ding) [inline src/version/Version.h:1]: 2020

File: src/version/Version.cpp.in [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
```
Comment (laura-ding) [inline src/version/Version.cpp.in:1]: 2020

Comment (yixinglu) [reply src/version/Version.cpp.in]: fixed

File: src/version/Version.h (hunk 1 above)
Comment (yixinglu) [reply src/version/Version.h]: fixed

**Review by laura-ding — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**